doc/md/ddl/sql-types.md

```diff
@@ -0,0 +1,322 @@
+  schema = schema.test
+  column "c1" {
+    type = bit
+  }
+  column "c2" {
+    type = bit(4)
+  }
+}
+```
+
+### Binary
+
+The `varbinary` and `binary` types allow storing binary byte strings.
+
+```hcl
+table "t" {
+  schema = schema.test
+  column "c1" {
+    // Equals to binary(1).
+    type = binary
+  }
+  column "c2" {
+    type = binary(10)
+  }
+  column "c3" {
+    type = varbinary(255)
+  }
+}
+```
+
+### Blob
+
+The `tinyblob`, `mediumblob`, `blob` and `longblob` types allow storing binary large object.
+
+```hcl
+table "t" {
+  schema = schema.test
+  column "c1" {
+    type = tinyblob
+  }
+  column "c2" {
+    type = mediumblob
+  }
+  column "c3" {
+    type = blob
+  }
+  column "c4" {
+    type = longblob
+  }
+}
+```
+
+### Boolean
+
+The `bool` and `boolean` are mapped to `tinyint(1)` in MySQL. Still, Atlas allows maintaining columns of type `bool`
+in the schema for simplicity reasons.
+
+```hcl
+table "t" {
+  schema = schema.test
+  column "c1" {
+    type = bool
+  }
+  column "c2" {
+    type = boolean
+  }
+}
+```
+
+Learn more about the motivation for these types in
+[MySQL website](https://dev.mysql.com/doc/refman/8.0/en/other-vendor-data-types.html).
+
+### Date and Time
+
+Atlas supports the standard MySQL types for storing date and time values. `time`, `timestamp`, `date`, `datetime`,
+and `year`.
+
+```hcl
+table "t" {
+  schema = schema.test
+  column "c1" {
+    type = time
+  }
+  column "c2" {
+    type = timestamp
+  }
+  column "c3" {
+    type = date
+  }
+  column "c4" {
+    type = datetime
+  }
+  column "c5" {
+    type = year
+  }
+  column "c6" {
+    type = time(1)
+  }
+  column "c7" {
+    type = timestamp(2)
+  }
+  column "c8" {
+    type = datetime(4)
+  }
+}
+```
+
+### Fixed Point
+
+The `decimal` and `numeric` types are supported for storing exact numeric values. Note that, in MySQL the two types
+are identical.
+
+```hcl
+table "t" {
+  schema = schema.test
+  column "c1" {
+    // Equals to decimal(10) as the
+    // default precision is 10.
+    type = decimal
+  }
+  column "c2" {
+    // Equals to decimal(5,0).
+    type = decimal(5)
+  }
+  column "c3" {
+    type = decimal(5,2)
+  }
+  column "c4" {
+    type     = numeric
+    unsigned = true
+  }
+}
+```
+
+### Floating Point
+
+The `float` and `double` types are supported for storing approximate numeric values.
+
+```hcl
+table "t" {
+  schema = schema.test
+  column "c1" {
+    type = float
+  }
+  column "c2" {
+    type = double
+  }
+  column "c3" {
+    type     = float
+    unsigned = true
+  }
+  column "c4" {
+    type     = double
+    unsigned = true
+  }
+}
+```
+
+### Enum
+
+The `enum` type allows storing a set of enumerated values.
+
+```hcl
+table "t" {
+  schema = schema.test
+  column "c1" {
+    type = enum("a", "b")
+  }
+  column "c2" {
+    type = enum(
+      "c",
+      "d",
+    )
+  }
+}
+```
+
+### Integer
+
+The `int`, `tinyint`, `smallint`, `mediumint`, `bigint` integer types are support by Atlas.
```

Comment:
```suggestion
The `tinyint`, `smallint`, `int`, `mediumint`, `bigint` integer types are support by Atlas.
```

Reply:
Switched the order to based on size because you did the same with blobs, so change the order in the columns below as well.